xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/XWikiLivedata.vue

```diff
@@ -47,10 +47,20 @@
 // eslint-disable-next-line camelcase
 __webpack_public_path__ = window.liveDataBaseURL;
 
+import Vue from "vue";
+import VueTippy from "vue-tippy";
+import 'tippy.js/themes/light-border.css'
 import LivedataAdvancedPanels from "./panels/LivedataAdvancedPanels.vue";
 import LivedataLayout from "./layouts/LivedataLayout.vue";
 import LivedataPersistentConfiguration from "./LivedataPersistentConfiguration.vue";
 
+// Declare vue plugins here
```

Comment:
Could we consider using the webjar? https://search.maven.org/artifact/org.webjars.npm/tippy.js

Reply:
This is the webjar for tippy.js, not vue-tippy.
If you use Tippy.js, then you'll have to create the vue wrapper by yourself.

Reply:
indeed, too bad.